core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -305,6 +305,7 @@ enum class UniqueType(
     Unsellable("Unsellable", UniqueTarget.Building),
     ObsoleteWith("Obsolete with [tech]", UniqueTarget.Building, UniqueTarget.Resource, UniqueTarget.Improvement),
     IndicatesCapital("Indicates the capital city", UniqueTarget.Building),
+    MovedToNewCapital("Moved to the new capital", UniqueTarget.Building),
```

Comment:
Unclear phrasing - "Moved" can be passive, or past-tense. Better to use "Moves", and under what condition - so all in all "Moves to new capital when capital changes".